packages/aws-rfdk/lib/deadline/scripts/python/configure_identity_registration_settings.py

```diff
@@ -420,7 +406,7 @@ def get_rfdk_registration_settings(dl_secrets: DeadlineSecretsCommandClient) ->
 
 
 def delete_setting(dl_secrets: DeadlineSecretsCommandClient, setting: LoadBalancerIdentityRegistrationSetting) -> None:
-    print(dl_secrets.dry_run([
+    print(dl_secrets.dry_run([ # TODO: Why is this dry run?
```

Comment:
Should this actually run the command instead of being a dry run?

Reply:
yep - good catch, this looks like it was a left-over from debugging.